DESCRIPTION

```diff
@@ -4,11 +4,14 @@ Title: R Interface to TopoToolbox
 Authors@R: c(person("Domenic", "Bartha", role = c("cre","aut"),
                      email = "bartha1@uni-potsdam.de"))
 Maintainer: Domenic Bartha <bartha1@uni-potsdam.de>
-Depends: R (>= 4.1.0)
+Depends: R (>= 4.1.0),
+	 terra
 License: GPL-3 + file LICENSE
 Description: Topotoolboxr provides an R interface to TopoToolbox, a set of tools for digital terrain analysis and landscape evolution modeling.
 Encoding: UTF-8
 LazyData: true
+Imports:
+    terra
```

Comment:
`terra` needs to be in `Depends` and not `Imports`, so these two lines need to be deleted (I think they crept back in when you merged your main branch). We get a warning about this in `R CMD check`.

Reply:
Of course it is done, i am really annoyed by git. I don't understand why so many changes are reversed or not truly pushed. I have changed this now for the 3. Time. 
But i guess, i am doing something wrong.